### DISCUSSION TITLE
请问作者有文章介绍这个插件吗

### DISCUSSION THREAD

**Discussion by jessecaii** [GreatAuk/plugin-web-update-notification#7] [General]
想知道实现的原理，搜了一圈没有搜到

**Comment by GreatAuk** [comment]
没有，可以直接看源码，很简单的。关键实现可以看这里 https://github.com/GreatAuk/plugin-web-update-notification/tree/master/packages/core/src
<img width="965" alt="image" src="https://user-images.githubusercontent.com/20253809/206405394-d1d16f87-58c6-4ed8-b52a-75f0bfd3a01f.png">
原理 README 上有写

**Reply by GreatAuk** [reply]
<img width="1112" alt="image" src="https://user-images.githubusercontent.com/20253809/206405838-0095be64-59b3-40ea-b959-a72a51cfd901.png">
这些是插件注入的内容。

**Comment by GreatAuk** [comment]
https://juejin.cn/post/7209234917288886331
